.github/workflows/release.yml

```diff
@@ -100,6 +100,19 @@ jobs:
             options+=('--tags' '${{ github.event.inputs.tag }}')
           fi
           gren release --draft "${options[@]}" --username=${{ github.repository_owner }} --repo=${{ github.event.repository.name }} --token=${{ secrets.GITHUB_TOKEN }}
+      - uses: azure/setup-helm@v1
+      - run: helm package helm-charts/move2kube
+      - run: mkdir temp/
+      - run: mv *.tgz temp/
+      - run: curl -L -o oldindex.yaml https://konveyor.io/move2kube/index.yaml
+      - run: helm repo index --merge oldindex.yaml --url https://github.com/konveyor/move2kube-operator/releases/download/${{ github.event.inputs.tag }} temp/
+      - uses: HarikrishnanBalagopal/upload-release-action@v3
```

Comment:
Can we move this repository to konveyor too? We are using in multiple places and its a useful reusable action for konveyor.